Doc/library/asyncio-task.rst

```diff
@@ -610,7 +610,9 @@ Running in Threads
    Asynchronously run function *func* in a separate thread.
 
    Any \*args and \*\*kwargs supplied for this function are directly passed
-   to *func*.
+   to *func*. Also, the current :class:`contextvars.Context` is propogated,
+   allowing context variables from the main thread to be accessed in the
```

Comment:
```suggestion
   allowing context variables from the event loop thread to be accessed in the
```

Reply:
Good point. It's most commonly going to be the main thread, but in multi-loop programs it may could very well be sharing context from another thread.